scout/server/blueprints/variants/static/form_scripts.js

```diff
@@ -1,69 +1,107 @@
-/* exported populateCytobands */
-function populateCytobands(cytobands){
-  var chromPosPattern = new RegExp("^(?:chr)?([1-9]|1[0-9]|2[0-2]|X|Y|MT)$");
-  var chrom = document.forms["filters_form"].elements["chrom"].value;
-  var chromPos = "";
-  if (typeof document.forms["filters_form"].elements["chrom_pos"] !== "undefined") {
-  	chromPos=document.forms["filters_form"].elements["chrom_pos"].value;
+/* exported sanitizeChromSelOptions() */
+function sanitizeChromSelOptions() {
+	var chrom = [];
+	var chrom_select = document.forms["filters_form"].elements["chrom"];
+  var options = chrom_select && chrom_select.options;
+  var opt;
+
+  // clear other options if All is selected
+	// and take note of otherwise selected chrs
+  for (var i=1, iLen=options.length; i<iLen; i++) {
+    opt = options[i];
+    if (opt.selected) {
+    	if (chrom_select.selectedIndex === 0) {
+				opt.selected = false
+			} else {
+    		chrom.push(opt.value);
+			}
+    }
+  }
+
+	console.log("Depopulate cytobands if more than one chrom is selected, all selected ,")
+	if (chrom.length > 1) {
+		// disable
+		for (elem of [cytoStart, cytoEnd]) {
+			elem.options.length = 0; //remove previous cytoband select options
+			startElem.value = null
+			endElem.value = null
+		}
+		return
+	}
+}
+
+function getSelectedChromosomes() {
+	var chrom_select = document.forms["filters_form"].elements["chrom"];
+
+	var chrom = [];
+  var options = chrom_select && chrom_select.options;
```

Comment:
You could simplify a lot the code from this line. Something like this perhaps?

```
if (chrom_select.selectedIndex === 0) {
		return chrom;
}

for (var i=0; i < chrom_select.selectedOptions.length; i++) {
		chrom.push(chrom_select.selectedOptions[i].value);
}
return chrom;
```

Reply:
The complication with that is that I would also have to add a check for the empty value `All` option: since it can be selected or not, I can't just assume I know its position in the selectedOptions.

Reply:
But you know it's position actually, it's always the first if it's selected, no?

Reply:
If you have: 
```
(chrom_select.selectedIndex === 0) {
		return chrom;
}
```

before that loop, whenever the first option (All) is selected, you will not even reach the loop after. Check the docs: https://developer.mozilla.org/en-US/docs/Web/API/HTMLSelectElement/selectedIndex

Reply:
That is correct, although it may not be super evident that the following requires it. But that can be said for a lot. Using  the `selectedOptions` collection and going from 0 does make that loop a little easier to read - let's go with that.

Reply:
> may not be super evident

You could add a comment perhaps?